doc/rados/operations/health-checks.rst

```diff
@@ -612,3 +612,15 @@ happen if they are misplaced or degraded (see *PG_AVAILABILITY* and
 You can manually initiate a scrub of a clean PG with::
 
   ceph pg deep-scrub <pgid>
+
+OSD_ALL_SCRUBS_OVERDUE
+----------------------
+
+If the ratio of overdue scrubs to total pgs is greater than
+``mon_warn_pg_scrubs_overdue_ratio`` this warning is triggered.
```

Comment:
Just a note: every health item here should have a paragraph that describe what the error means and why it is happening, and then another paragraph (or more) describing what can be done to mitigate the problem.  In this case that might be increasing the scrub interval, or getting faster OSDs, or making scrub more aggressive (reference the osd scrub related options like sleep and max scrubs, etc).